src/plugins/intel_cpu/src/transformations/snippets/x64/pass/mul_add_to_fma.cpp

```diff
@@ -2,10 +2,11 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <snippets/itt.hpp>
-
 #include "mul_add_to_fma.hpp"
-#include "snippets/snippets_isa.hpp"
+
+#include "snippets/itt.hpp"
+#include "snippets/utils.hpp"
```

Comment:
Same: not sure we need this header here after fallback to `copy_runtime_info`

Reply:
It's outdated change. Thanks